### PR TITLE
Add Async Download Methods for Annotations in Videos, Volumes, PCD

### PR DESCRIPTION
**Pull request by GoldenAnpu**

### CODE REVIEW

File: supervisely/api/video/video_annotation_api.py [hunk 1]
```diff
@@ -254,18 +252,28 @@ async def download_async(
         video_id: int,
         video_info=None,
         semaphore: Optional[asyncio.Semaphore] = None,
+        force_metadata_for_links: bool = True,
+        integer_coords: bool = True,
```
Comment (almazgimaev) [inline supervisely/api/video/video_annotation_api.py:256]: typing: Optional

File: supervisely/api/video/video_annotation_api.py [hunk 2]
```diff
@@ -254,18 +252,28 @@ async def download_async(
...
         Download information about VideoAnnotation by video ID from API asynchronously.
 
         :param video_id: Video ID in Supervisely.
         :type video_id: int
-        :param video_info: VideoInfo object. Use it to avoid additional request to the server.
+        :param video_info: Does not affect the result, but is left for compatibility with the method signature.
         :type video_info: VideoInfo, optional
         :param semaphore: Semaphore to limit the number of parallel downloads.
         :type semaphore: asyncio.Semaphore, optional
+        :param force_metadata_for_links: Force metadata for links.
```
Comment (almazgimaev) [inline supervisely/api/video/video_annotation_api.py:268]: not clear descriptions for `force_metadata_for_links` and `integer_coords` params.
(the same in the `download_bulk_async` method)

File: supervisely/api/video/video_annotation_api.py [hunk 3]
```diff
@@ -280,15 +288,71 @@ async def download_async(
...
             ann_info = loop.run_until_complete(api.video.annotation.download_async(video_id))
         """
-        if video_info is None:
-            video_info = self._api.video.get_info_by_id(video_id)
+        return await self.download_bulk_async(
+            [video_id],
+            semaphore,
+            force_metadata_for_links,
+            integer_coords,
+            progress_cb,
```
Comment (almazgimaev) [inline supervisely/api/video/video_annotation_api.py:296]: better to pass named args `video_ids=[video_id],`
(the same for pcd and volumes)

File: supervisely/api/video/video_annotation_api.py [hunk 4]
```diff
@@ -280,15 +288,71 @@ async def download_async(
...
+                }
+                response = await self._api.post_async(
+                    self._method_download_bulk,
+                    json=json_data,
+                )
+                if progress_cb is not None:
+                    progress_cb(len(batch))
+                return response.json()
+
+        tasks = [fetch_with_semaphore(batch) for batch in batched(video_ids)]
```
Comment (almazgimaev) [inline supervisely/api/video/video_annotation_api.py]: ?? set batch size 10 instead of the default 50

**Review by almazgimaev — APPROVED** [review]

File: supervisely/api/video/video_annotation_api.py (hunk 1 above)
Comment (GoldenAnpu) [reply supervisely/api/video/video_annotation_api.py:256]: Optional[X] is equivalent to Union[X, None]
That's not the case.

**Pull request merged**